lib/cucumber/configuration.rb

```diff
@@ -20,12 +22,14 @@ def self.default
     #
     # @param event_id [Symbol, Class, String] Identifier for the type of event to subscribe to
     # @param handler_object [Object optional] an object to be called when the event occurs
-    # @yield [Object] Block to be called when th event occurs
+    # @yield [Object] Block to be called when the event occurs
     # @method on_event
-    def_instance_delegator :event_bus, :register, :on_event
+    def_instance_delegator :event_bus, :on, :on_event
 
     # @private
-    def_instance_delegator :event_bus, :notify
+    def notify(message, *args)
+      event_bus.send(message, *args)
```

Comment:
The event bus does also have a `broadcast` method that takes an event class instance as argument. Should it also be made available through the configuration object?